common/edicts/01_campaigns.txt

```diff
@@ -0,0 +1,855 @@
+	resources = {
+		category = campaigns
+		upkeep = {
+			energy = @campaignCost
+			multiplier = value:edict_size_effect
+		}
+	}
+
+	modifier = {
+		pop_growth_speed = 0.10
+	}
+
+	potential = {
+		always = no
+	}
+
+	ai_weight = {
+		weight = @EdictHighPrio
+	}
+}
+
+###########
+# Machine Empire
+###########
+
+robot_campaign = {
+	length = @campaignDuration
+	icon = "GFX_edict_type_time"
+	resources = {
+		category = campaigns
+		upkeep = {
+			energy = @campaignCost
+			multiplier = value:edict_size_effect
+		}
+	}
+
+	modifier = {
+		planet_pop_assembly_mult = 0.10
+	}
+
+	potential = {
+		always = no
+	}
+
+	ai_weight = {
+		weight = @EdictHighPrio
+	}
+}
+
+machine_learning_campaign = {
+	length = @campaignDuration
+	icon = "GFX_edict_type_time"
+	resources = {
+		category = campaigns
+		upkeep = {
+			energy = @campaignCost
+			multiplier = value:edict_size_effect
+		}
+	}
+
+	modifier = {
+		species_leader_exp_gain = 0.25
+	}
+
+	potential = {
+		has_authority = auth_machine_intelligence
+		has_technology = tech_planetary_unification
+	}
+
+	ai_weight = {
+		weight = 0
+	}
+}
+
+#################
+# Hive Mind
+#################
+
+drone_campaign = {
+	length = @campaignDuration
+	icon = "GFX_edict_type_time"
+	resources = {
+		category = campaigns
+		upkeep = {
+			trigger = {
+				is_lithoid_empire = no
+			}
+			food = @hiveFood
+			multiplier = value:edict_size_effect
+		}
+		upkeep = {
+			trigger = {
+				is_lithoid_empire = yes
+			}
+			minerals = @hiveFood
+			multiplier = value:edict_size_effect
+		}
+	}
+
+	modifier = {
+		pop_growth_speed = 0.10
+	}
+
+	potential = {
+		always = no
+	}
+
+	ai_weight = {
+		weight = @EdictHighPrio
+	}
+}
+
+learning_campaign = {
+	length = @campaignDuration
+	icon = "GFX_edict_type_time"
+	resources = {
+		category = campaigns
+		upkeep = {
+			trigger = {
+				is_lithoid_empire = no
+			}
+			food = @hiveFood
+			multiplier = value:edict_size_effect
+		}
+		upkeep = {
+			trigger = {
+				is_lithoid_empire = yes
+			}
+			minerals = @hiveFood
+			multiplier = value:edict_size_effect
+		}
+	}
+
+	modifier = {
+		species_leader_exp_gain = 0.25
+	}
+
+	potential = {
+		has_authority = auth_hive_mind
+		has_technology = tech_planetary_unification
+	}
+
+	ai_weight = {
+		weight = 0
+	}
+}
+
+war_drone_campaign = {
+	length = @campaignDuration
+	icon = "GFX_edict_type_time"
+	resources = {
+		category = campaigns
+		upkeep = {
+			trigger = {
+				is_lithoid_empire = no
+			}
+			food = @hiveFood
+			multiplier = value:edict_size_effect
+		}
+		upkeep = {
+			trigger = {
+				is_lithoid_empire = yes
+			}
+			minerals = @hiveFood
+			multiplier = value:edict_size_effect
+		}
+	}
+
+	modifier = {
+		army_damage_mult = 0.15
+	}
+
+	potential = {
+		has_authority = auth_hive_mind
+		has_technology = tech_planetary_unification
+	}
+
+	ai_weight = {
+		weight = 0
+	}
+}
+
+
+##########################################################################
+# Rare resource edicts
```

Comment:
Can probably comment these out for now pending the completion of the new resources full implementation.